src/client/src/App.jsx

```diff
@@ -1,28 +1,117 @@
-import "./App.css";
 import { Routes, Route, BrowserRouter as Router, } from "react-router-dom";
 import React, {useEffect, useState} from "react";
+import { FlashCardPage } from "./FlashCard/Page";
 
-const Placeholder = () => {
-    const [users, setUsers] = useState([]);
 
-    useEffect(() => {
-        fetch('/api/users').then( async (user) => {
-            const usersReturned = await user.json();
-            setUsers(Object.values(usersReturned)[0])
-        })
-    }, []);
+const DUMMY_FLASH_CARDS = [
+  {
+    id: 1,
+    term: 'andrew',
+    definition: 'spencer',
+  },
+];
 
-    return <div>{users.join(',')}</div>
-};
+// Make sure this is unique!
+let nextId = 2;
 
-function App() {
+/**
+ * 
+ * @param {*} props onSubmit function
+ */
+function FlashCardForm(props) {
```

Comment:
This should be moved to its own file, and refactored to be split up - separate the concerns.
API hook and state management should also be extracted - Move to `react-query`